package.json

```diff
@@ -1,132 +1,126 @@
-    "precommit": "lint-staged",
-    "postcommit": "git reset",
-    "perf": "npm run build && node --expose-gc scripts/perf.js",
-    "perf:init": "node scripts/perf-init.js",
-    "perf:debug": "node --inspect-brk scripts/perf-debug.js"
-  },
-  "repository": "rollup/rollup",
-  "keywords": [
-    "modules",
-    "bundler",
-    "bundling",
-    "es6",
-    "optimizer"
-  ],
-  "author": "Rich Harris",
-  "contributors": [
-    "Oskar Segersvärd <victorystick@gmail.com>",
-    "Bogdan Chadkin <trysound@yandex.ru>"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/rollup/rollup/issues"
-  },
-  "homepage": "https://github.com/rollup/rollup",
-  "dependencies": {
-    "@types/estree": "0.0.39",
-    "@types/node": "*"
-  },
-  "devDependencies": {
-    "@types/acorn": "^4.0.3",
-    "@types/chokidar": "^1.7.5",
-    "@types/minimist": "^1.2.0",
-    "@types/pretty-ms": "^3.0.0",
-    "acorn": "^5.7.1",
-    "acorn-dynamic-import": "^3.0.0",
-    "acorn-import-meta": "^0.2.1",
-    "ansi-escapes": "^3.1.0",
-    "buble": "^0.19.3",
-    "chokidar": "^2.0.4",
-    "console-group": "^0.3.1",
-    "date-time": "^2.1.0",
-    "eslint": "^5.3.0",
-    "eslint-plugin-import": "^2.13.0",
-    "execa": "^0.10.0",
-    "fixturify": "^0.3.4",
-    "hash.js": "^1.1.5",
-    "husky": "^0.14.3",
-    "immutable": "^3.8.2",
-    "is-reference": "^1.1.0",
-    "istanbul": "^0.4.3",
-    "lint-staged": "^7.2.0",
-    "locate-character": "^2.0.5",
-    "magic-string": "^0.25.0",
-    "minimist": "^1.2.0",
-    "mocha": "^5.2.0",
-    "prettier": "^1.14.0",
-    "pretty-bytes": "^5.1.0",
-    "pretty-ms": "^3.2.0",
-    "remap-istanbul": "^0.11.1",
-    "require-relative": "^0.8.7",
-    "rollup": "^0.63.5",
-    "rollup-plugin-buble": "^0.19.2",
-    "rollup-plugin-commonjs": "^9.1.4",
-    "rollup-plugin-json": "^3.0.0",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-string": "^2.0.2",
-    "rollup-plugin-typescript": "^0.8.1",
-    "rollup-pluginutils": "^2.3.0",
-    "sander": "^0.6.0",
-    "shx": "^0.3.2",
-    "signal-exit": "^3.0.2",
-    "source-map": "^0.6.1",
-    "source-map-support": "^0.5.6",
-    "sourcemap-codec": "^1.4.1",
-    "tslint": "^5.11.0",
-    "turbocolor": "^2.4.1",
-    "typescript": "^3.0.1",
-    "uglify-js": "^3.4.6",
-    "url-parse": "^1.4.3"
-  },
-  "files": [
-    "dist/rollup.browser.js",
-    "dist/rollup.es.js",
-    "dist/rollup.js",
-    "dist/**/*.d.ts",
-    "bin/rollup",
-    "README.md"
-  ],
-  "lint-staged": {
-    "{src,bin,browser,typings}/**/*.ts": [
-      "tslint --project . --fix",
-      "prettier --write",
-      "git add"
-    ],
-    "test/**/*.ts": [
-      "tslint --project test/typescript --fix",
-      "prettier --write",
-      "git add"
-    ]
-  }
+	"name": "rollup",
+	"version": "0.64.1",
+	"description": "Next-generation ES module bundler",
```

Comment:
Could you not do this prettier change?

Reply:
WIP :) this is not yet ready for review. Please hold off until that tag is removed.

Reply:
@TrySound it's ready 🎉